docs/source/app-dev/bindings-java/codegen.rst

```diff
@@ -105,43 +105,53 @@ Map DAML primitives to Java types
+| DAML type                      | Java type                                  | Java Bindings          |
+|                                |                                            | Value Type             |
++================================+============================================+========================+
+| ``Int``                        | ``java.lang.Long``                         | `Int64`_               |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Decimal``                    | ``java.math.BigDecimal``                   | `Decimal`_             |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Text``                       | ``java.lang.String``                       | `Text`_                |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Bool``                       | ``java.util.Boolean``                      | `Bool`_                |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Party``                      | ``java.lang.String``                       | `Party`_               |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Date``                       | ``java.time.LocalDate``                    | `Date`_                |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Time``                       | ``java.time.Instant``                      | `Timestamp`_           |
++--------------------------------+--------------------------------------------+------------------------+
+| ``List`` or ``[]``             | ``java.util.List``                         | `DamlList`_            |
++--------------------------------+--------------------------------------------+------------------------+
+| ``TextMap``                    | ``java.util.Map``                          | `TextMap`_             |
+|                                | Restricted to using ``String`` keys.       |                        |
++--------------------------------+--------------------------------------------+------------------------+
+| ``Optional``                   | ``java.util.Optional``                     | `DamlOptional`_        |
++--------------------------------+--------------------------------------------+------------------------+
+| ``()`` (Unit)                  | **None** since the Java language doesn’t   | `Unit`_                |
+|                                | have a direct equivalent of DAML’s Unit    |                        |
+|                                | type ``()``, the generated code uses the   |                        |
+|                                | Java Bindings value type.                  |                        |
```

Comment:
Any particular reason for removing the reference to the specific type?

Reply:
I didn't want to duplicate the reference in the same row as referring to the 'Java Bindings value type' which is the title on the third column that contains the reference should suffice?

Reply:
Oh, I see, I've misread this table! My bad.

I wonder if you might prefer to use a [list table](http://docutils.sourceforge.net/docs/ref/rst/directives.html#list-table) for this? Might be easier to read and write.